src/sqltest/parts/notebook/model/cell.test.ts

```diff
@@ -0,0 +1,262 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Can you give us an idea how much of this was lifted from the other repo and how much needed to be refactored/rewritten? Would help to understand what should be looked at here :).

Reply:
Everything is from the other repo. Just recompiled.